app/view/LayerTree.js

```diff
@@ -91,8 +85,9 @@ Ext.define('CpsiMapview.view.LayerTree', {
 
         var mapComp = BasiGX.util.Map.getMapComponent();
         var map = mapComp && mapComp.getMap();
-        if (map) {
-            me.setStore(me.makeLayerStore(map));
+        me.map = map;
```

Comment:
Minor: why not set `me.map = mapComp && mapComp.getMap()` directly?

Reply:
Correct, could be set directly. I'll change that